Source/WebKit/UIProcess/WebPageProxyInternals.h

```diff
@@ -59,7 +59,9 @@
 #endif
 
 #if ENABLE(DRAG_SUPPORT)
+#include "SandboxExtension.h"
 #include <WebCore/DragActions.h>
+#include <WebCore/DragData.h>
```

Comment:
I don't think the include is necessary here? Add it in implementation files if necessary.